src/crimson/os/seastore/segment_manager/ephemeral.cc

```diff
@@ -177,12 +175,9 @@ EphemeralSegmentManager::init_ertr::future<> EphemeralSegmentManager::init()
     return crimson::ct_error::invarg::make();
   }
 
-  void* addr = ::mmap(
-    nullptr,
-    config.size,
-    PROT_READ | PROT_WRITE, MAP_SHARED | MAP_ANONYMOUS,
-    -1,
-    0);
+  // memset 0 is not needed: anonymous mapping is zero-filled
+  void* addr = ::mmap(nullptr, config.size, PROT_READ | PROT_WRITE,
+                      MAP_PRIVATE | MAP_ANONYMOUS, -1, 0);
```

Comment:
Tests look good though I would like additional look here before merging, are we ok with switching to anonymous memory mapping (MAP_PRIVATE)?

@athanatos, @xxhdx1985126

Reply:
I understand the change, thanks for elaborating. AFAIK there's no IPC requirement and this should be ok (also seen in the tests). However, I would still want an additional look to make sure there aren't any nuances which I'm not aware of.

Reply:
No worries :)

Reply:
@chanyoung FYI we actually have two crimson standups on the calendar for different timezones -- "Crimson Weekly" and "Discussion of Crimson/Seastar-OSD development."  Perhaps the other one would work better for you?  If not, of course, no worries and we can certainly communicate here.  Thanks for the contributions!